src/components/shared/pricing/PricingSection.tsx

```diff
@@ -72,17 +73,17 @@ export const PricingSection = ({
   onBusinessSwitchToggled,
   onCheckoutButtonClicked,
   onBusinessPlansSelected,
+  darkMode,
 }: PriceTableProps): JSX.Element => {
   const banner = require('@/assets/lang/en/banners.json');
 
   const isBusiness = activeSwitchPlan === 'Business';
-  const labelDiscount = isBusiness ? '10' : '23';
+  const labelDiscount = '15';
   const showLoadingCards = loadingCards;
   const showBusinessCards = isBusiness && !loadingCards && !!businessBillingFrequency;
   const isIndividual = activeSwitchPlan === 'Individuals' || activeSwitchPlan === 'Lifetime';
   const showIndividualCards = isIndividual && !loadingCards;
-
-  const showSwitchComponent = activeSwitchPlan === 'Business' && !hideBusinessCards;
+  const showSwitchComponent = !loadingCards && !hideSwitchSelector && activeSwitchPlan !== 'Lifetime';
```

Comment:
Same here.

Reply:
I understood that the monthly plans also had to be shown in individuals. But not